spec/fixtures/vcr_cassettes/cancel_visit.yml

```diff
@@ -8,7 +8,7 @@ http_interactions:
       string: '{"prison_id":"0614760e-a773-49c0-a29c-35e743e72555","prisoner":{"first_name":"Oscar","last_name":"Wilde","date_of_birth":"1980-12-31","number":"a1234bc"},"visitors":[{"first_name":"Ada","last_name":"Lovelace","date_of_birth":"1970-11-30"},{"first_name":"Charlie","last_name":"Chaplin","date_of_birth":"2005-01-02"}],"contact_email_address":"ada@test.example.com","contact_phone_no":"01154960222","slot_options":["2016-07-29T13:30/14:30","2016-07-29T14:45/15:45","2016-07-30T09:45/11:15"]}'
     headers:
       User-Agent:
-      - excon/0.55.0
+      - excon/0.56.0
```

Comment:
True bit not worth reverting the changes as some of the VCR cassette files have been regenerated